instrument/instrument.go

```diff
@@ -75,7 +75,7 @@ func ObserveWithExemplar(ctx context.Context, histogram prometheus.Observer, sec
 	if traceID, ok := tracing.ExtractSampledTraceID(ctx); ok {
 		histogram.(prometheus.ExemplarObserver).ObserveWithExemplar(
 			seconds,
-			prometheus.Labels{"traceID": traceID},
+			prometheus.Labels{"trace_id": traceID},
```

Comment:
I guess this isn't a huge deal since exemplars are short-lived and only in memory but should we send both for a period of time?

Reply:
That does seem like an attractive idea.